serverless/src/getContactInfo/__tests__/getCmrPreferencesData.test.js

```diff
@@ -0,0 +1,39 @@
+import nock from 'nock'
+import { v4 as uuidv4 } from 'uuid'
+import * as getClientId from '../../../../sharedUtils/getClientId'
+import * as getEarthdataConfig from '../../../../sharedUtils/config'
+import { getCmrPreferencesData } from '../getCmrPreferencesData'
+
+jest.mock('uuid')
+uuidv4.mockImplementation(() => 'mock-request-id')
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('getCmrPreferencesData', () => {
+  test('correctly requests a users data from cmr-ordering', async () => {
+    jest.spyOn(getEarthdataConfig, 'getEarthdataConfig').mockImplementation(() => ({ cmrHost: 'https://cmr.com' }))
+    jest.spyOn(getClientId, 'getClientId').mockImplementation(() => ({ lambda: 'eed-edsc-test-serverless-lambda' }))
+
+    nock(/cmr/)
+      .matchHeader('Authorization', 'Bearer fake.access.token')
+      .matchHeader('Client-Id', 'eed-edsc-test-serverless-lambda')
+      .matchHeader('X-Request-Id', 'mock-request-id')
+      .post(/ordering\/api/)
+      .reply(200, {
+        user: {
+          mock: 'cmr'
+        }
+      })
+
+    const cmrData = await getCmrPreferencesData('urs_user', 'fake.access.token', 'test')
+
+    const { data: preferences } = cmrData
+    expect(preferences).toEqual({
```

Comment:
I'm not sure if this expect is correct. Looking at the usage of the value returned from `getCmrPreferencesData`, I feel like maybe it is missing one of the `data`s, but I'm not 100% sure.

But I think this test can be better by testing the full response of the method. In the handler you are using a status, and the axios data object. The test here should be proving that the whole object is returned as expected, not just the value from cmr-ordering